cmd/frontend/internal/app/updatecheck/client.go

```diff
@@ -185,6 +188,66 @@ func getAndMarshalAggregatedUsageJSON(ctx context.Context) (_ json.RawMessage, _
 	return serializedCodeIntelUsage, serializedSearchUsage, nil
 }
 
+func getDependencyVersions(ctx context.Context) (json.RawMessage, error) {
+	var (
+		err error
+		dv  dependencyVersions
+	)
+	// get redis cache server version
+	dv.RedisCacheVersion, err = getRedisVersion(redispool.Cache.Dial)
+	if err != nil {
+		log15.Warn("unable to connect to redis cache instance", "error", err)
+	}
+
+	// get redis store server version
+	dv.RedisStoreVersion, err = getRedisVersion(redispool.Store.Dial)
+	if err != nil {
+		log15.Warn("unable to get redis store version", "error", err)
+	}
+
+	// get postgres version
+	err = dbconn.Global.QueryRowContext(ctx, "SELECT version();").Scan(&dv.PostgresVersion)
```

Comment:
No need for semi-colon.

```suggestion
	err = dbconn.Global.QueryRowContext(ctx, "SELECT version()").Scan(&dv.PostgresVersion)
```